### PR TITLE
Update webcam_demo.cpp

### PR DESCRIPTION
**Pull request by sturkmen72**
fix small bug

### CODE REVIEW

File: modules/text/samples/webcam_demo.cpp [hunk 1]
```diff
@@ -234,7 +234,8 @@ int main(int argc, char* argv[])
 
 
         frame.copyTo(out_img);
-        float scale_img  = (float)(600.f/frame.rows);
+        int scale = downsize ? 2 : 1;
+        float scale_img  = (float)((600.f/frame.rows)/scale);
         float scale_font = (float)(2-scale_img)/1.4f;
```
Comment (sturkmen72) [inline modules/text/samples/webcam_demo.cpp]: scale_font still need a better calculation

**Comment by sturkmen72** [comment]
@lluisgomez could you check it.

**Comment by lluisgomez** [comment]
Sure! Makes text readable when image is downsized.
Thanks!.

**Comment by mshabunin** [comment]
:+1:

**Pull request merged**